src/wallet/api.rs

```diff
@@ -52,6 +52,22 @@ use super::{
 
 const HARDENDED_DERIVATION: &str = "m/84'/1'/0'";
 
+/// Salt used for key derivation from a user-provided passphrase.
+const PBKDF2_SALT: &[u8; 8] = b"coinswap";
+/// Number of PBKDF2 iterations to strengthen passphrase-derived keys.
+///
+/// In production, this is set to **600,000 iterations**, following
```

Comment:
This seems like old code. this is now moved elsewhere. So change set can be reverted.